### PR TITLE
Using intelaipg/intel-optimized-tensorflow:1.9.0-mkl-py3 images across the board - part 1

### PR DESCRIPTION
**Pull request by ashahba**
## Reviewer Checklist

 - [ ] Do you understand it?
 - [ ] Is it correct?
 - [ ] Is it safe?
 - [ ] Is it legally compliant?
 - [ ] Is it robust?
 - [ ] Is it simple?
 - [ ] Is it tested?
 - [ ] Is it documented?
 - [ ] Is the style consistent?

See [the reviewer guide](docs/reviews.md) for more detail on each check.

### CODE REVIEW

File: mlt-templates/tf-dist-mnist/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Please don't add tensorflow here since it's being installed by the Docker
+# Installing Tensorflow due to: https://github.com/tensorflow/tensorflow/issues/20833
+tensorflow==1.8.0
```
Comment (s1113950) [inline mlt-templates/tf-dist-mnist/requirements.txt]: You're using the intel tensorflow image for this though. On the other templates you say to not install tensorflow here. Why is this any different?

Comment (ashahba) [reply mlt-templates/tf-dist-mnist/requirements.txt]: Yes, for now. But this is why:
https://github.com/kubeflow/tf-operator/issues/774

Comment (s1113950) [reply mlt-templates/tf-dist-mnist/requirements.txt]: So then does it make sense to not start from the Intel image and just use python3.6 for now?

Comment (ashahba) [reply mlt-templates/tf-dist-mnist/requirements.txt]: I have no strong opinion there, but I think using `Intel` images that has all the pieces in place including `MKL` and `Tensorflow` which already works with other templates is nice and reduces custom configurations required if we were to enable `MKL` ourselves.

**Comment by karthikvadla** [comment]
Really good to have :)

File: mlt-templates/tf-dist-mnist/requirements.txt (hunk 1 above)
Comment (s1113950) [reply mlt-templates/tf-dist-mnist/requirements.txt]: 👍 I'm cool with that

**Pull request merged**